libs/designsystem/src/lib/components/fab-sheet/fab-sheet.component.ts

```diff
@@ -1,21 +1,21 @@
+import { DOCUMENT } from '@angular/common';
 import {
+  AfterContentInit,
+  AfterViewInit,
   Component,
   ContentChild,
+  ElementRef,
   HostBinding,
+  Inject,
   Input,
-  AfterContentInit,
-  AfterViewInit,
-  ViewChild,
-  ElementRef,
   Renderer2,
-  Inject,
+  ViewChild,
 } from '@angular/core';
-import { DOCUMENT } from '@angular/common';
 import { IonFab, IonFabButton, IonIcon } from '@ionic/angular';
 
-import { ActionSheetComponent } from '../modal/action-sheet/action-sheet.component';
-import { kirbyIconSettings } from '../icon/kirby-icon-settings';
 import { Icon } from '../icon/icon-settings';
+import { kirbyIconSettings } from '../icon/kirby-icon-settings';
+import { ActionSheetComponent } from '../modal/action-sheet/action-sheet.component';
```

Comment:
Great. Do we have any control over how imports are sorted?

Reply:
Yes, I thinks this is auto-fixed by the [ordered imports](https://palantir.github.io/tslint/rules/ordered-imports/) tslint rule, so we have some config options.

Reply:
Hmmm, if it's not a stylelint or prettier rule, how come the file hasn't been reformatted before 🤔? Perhaps no-one has modified the file ever 😁